### PR TITLE
add pympler

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@conda-forge/core Ready for review!

File: recipes/pympler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+{% set sha256 = "74d26dfb6f02777a5087be181ef7f8ae57625e7ec3ae41e2fdd24f31289e772e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+#  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/pympler/meta.yaml]: Unless you are planning to use this soon please remove the commented line.

**Comment by ocefpaf** [comment]
One minor comment that can be addressed in the feedstock.

**Comment by CJ-Wright** [comment]
@ocefpaf Thank you very much!

**Pull request merged**